coops-ui/src/resources/business-config.ts

```diff
@@ -0,0 +1,66 @@
+export class Flow {
+  feeCode: string
+  displayName: string
+  certifyText: string
+}
+
+export class Business {
+     typeEnum: string
+     displayName: string
+     flows: Array<Flow>
+}
+
+export const configJson = [{
+  typeEnum: 'BC',
+  displayName: 'Benefits Company',
+  flows: [
+    {
+      feeCode: 'OTADD',
```

Comment:
Did you want to use the fee code enum in place of these strings?

Reply:
you would think so, but keeping in mind this is meant to be an external resource and has no references to any of the front end code. I.E - This could also live in a plain text/json file, or in a database...think of it more as a table than application source code, as it has no dependency on any other source code (front or back end)